{
	"description": "The python-dns package contains the dnslib module that implements a DNS client and additional modules that define certain symbolic constants used by DNS, such as dnstype, dnsclass and dnsopcode.  \n\nSecurity Fix(es):  \n\n  * dnspython: denial of service in stub resolver (CVE-2023-29483)\n\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.  \n\nAdditional Changes:  \n\nFor detailed information on changes in this release, see the AlmaLinux Release Notes linked from the References section.\n",
	"id": "ALSA-2024:9423",
	"issued_date": 1731369600,
	"modules": [],
	"packages": [
		{
			"arch": "noarch",
			"checksum": "b0119d64551f2546511b878ac9aea12e3caf90ae97e450091c643fe02d410681",
			"checksum_type": "sha256",
			"epoch": "0",
			"filename": "python3-dns-2.6.1-3.el9.noarch.rpm",
			"name": "python3-dns",
			"reboot_suggested": 0,
			"release": "3.el9",
			"src": "python-dns-2.6.1-3.el9.src.rpm",
			"version": "2.6.1"
		}
	],
	"references": [
		{
			"href": "https://access.redhat.com/errata/RHSA-2024:9423",
			"id": "RHSA-2024:9423",
			"type": "rhsa"
		},
		{
			"href": "https://access.redhat.com/security/cve/CVE-2023-29483",
			"id": "CVE-2023-29483",
			"type": "cve"
		},
		{
			"href": "https://bugzilla.redhat.com/2274520",
			"id": "2274520",
			"type": "bugzilla"
		},
		{
			"href": "https://errata.almalinux.org/9/ALSA-2024-9423.html",
			"id": "ALSA-2024:9423",
			"type": "self"
		}
	],
	"severity": "moderate",
	"title": "Moderate: python-dns security update",
	"type": "security",
	"updated_date": 1731930544
}
